### DISCUSSION TITLE
Clarify difference between 'Move to available' and 'Resend message'

### DISCUSSION THREAD

**Discussion by Laurens-makel** [frankframework/frankframework#5878] [Q&A]
When you use a `JdbcTableListener`, you have two options available to deal with messages with an error status. You could select the message and press the `resend selected` button or you could press the `move to available` button.

To our understanding, the difference between those two is that `move to available` modifies the status column of the message and resets the value to `statusValueAvailable`, which causes the listener to process the message as if it was a completely new message, whilst `resend` just re-submits the message and sets the `isRetry` flag in the session.

Basically, `move to available` means 'pretend this message is completely new' and `resend` message means 'retry this message'.

Could you clarify what the exact differences are? How does this relate to the `maxRetries` attribute?

![image](https://github.com/frankframework/frankframework/assets/37303445/b8cc6c95-3440-41fb-89a3-0fca50e8a269)

**Comment by nielsm5** [comment]
Hi,

That's pretty much it! Depending on your Listener configuration it will process it as a new message, but might use the same MessageId or CorrelationId. This can be particularly useful if you want to re-process large datasets, 'the normal way'.

It is also possible in certain cases to place message in a 'HOLD' state. Which can then be made available at a later time.

A manual retry will do a duplicate check (with messagelog) but will skip the ProblematicHistoryCheck (otherwise always enabled).
